packages/attribute-registry-consumer/package.json

```diff
@@ -6,6 +6,8 @@
   "main": "dist",
   "type": "module",
   "scripts": {
+    "test": "vitest",
```

Comment:
is this still useful? do we have non-integration tests?

Reply:
ok, but keep in mind that if we include routes into tests, IMO it's worth considering the complete test starting from the routes instead of services, so they would be included in the integration tests

Reply:
btw if this is not currently used, we can remove this script and add it only if required

Reply:
Updated